packages/devtools_app/lib/src/screens/debugger/codeview.dart

```diff
@@ -1126,19 +1126,41 @@ class ScriptPopupMenuOption {
 }
 
 final defaultScriptPopupMenuOptions = [
-  copyScriptNameOption,
+  copyPackageOption,
+  copyFilePathOption,
   goToLineOption,
   openFileOption,
 ];
 
-final copyScriptNameOption = ScriptPopupMenuOption(
-  label: 'Copy filename',
+final copyPackageOption = ScriptPopupMenuOption(
+  label: 'Copy package path',
   icon: Icons.content_copy,
   onSelected: (_, controller) => Clipboard.setData(
     ClipboardData(text: controller.scriptLocation.value?.scriptRef.uri),
   ),
 );
 
+final copyFilePathOption = ScriptPopupMenuOption(
+  label: 'Copy file path',
+  icon: Icons.content_copy,
+  onSelected: (_, controller) async {
+    String? filePath;
+    final packagePath = controller.scriptLocation.value!.scriptRef.uri;
+    if (packagePath != null) {
+      final results = await serviceManager.service!.lookupResolvedPackageUris(
```

Comment:
can we use ResolvedUriManager instead so that we can pull this from the cache if possible?